src/media-popup.js

```diff
@@ -1,21 +1,33 @@
 // FIXME: Refactor later
-var ipcRenderer = require('electron').ipcRenderer;
+import { ipcRenderer }        from 'electron';
+
+var hls = require('hls.js');
 var img = document.getElementById('loadedImage');
 var video = document.getElementById('loadedVideo');
+var body = document.getElementsByTagName("body")[0];
+var hlsMedia = new hls();
 var mediaWidth = 0,
 mediaHeight = 0,
 maxWidth = 800,
 maxHeight = 600;
 
+if (process.platform !== 'darwin'){
+  body.style = "padding-top: 0px;"
+}
+
 function resizeMedia(media) {
   if (media.path[0].id === img.id) {
     mediaWidth = this.width;
     mediaHeight = this.height;
   } else {
-    mediaWidth = this.videoWidth;
-    mediaHeight = this.videoHeight;
+    if (video.title === "HLS media"){
+      mediaWidth = this.videoWidth * 4;
+      mediaHeight = this.videoHeight * 4;
+    } else {
+      mediaWidth = this.videoWidth
+      mediaHeight = this.videoHeight
```

Comment:
Hmm ... good question I just don't think about it in Javascript since the compiler doesn't seem to care about it. I usually do it, just not in Javascript. Will fix.